lumicks/pylake/force_calibration/tests/test_touchdown.py

```diff
@@ -70,12 +70,12 @@ def test_sine_with_polynomial(amplitude, frequency, phase_shift, poly_coeffs):
 
 
 def test_touchdown(mack_parameters):
-    stage_positions = np.arange(99.5, 103.5, 0.01)
+    stage_positions = np.arange(99.5, 103.5, 0.01 / 1502)
```

Comment:
The `/ 1502` factor seems a bit too magical for future readers. Can you maybe clarify it somehow? A named variable, comment, or a named function to generate the test data?